tools/submit.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+if [[ $# < 1 ]]
+then
+    echo "Usage: ./submit.sh JOB, where JOB is the job script."
+    exit 1
+fi
+
+ERR=0
+
+BIN=$(sed -n 's/BIN\s*=\s*//p' < $1)
+CFG=$(sed -n 's/CFG\s*=\s*//p' < $1)
+
+if ! [[ -f $BIN ]]
+then
+   echo "Binary $BIN not found!"
+   exit 1
+fi
+
+if ! [[ -f $BIN ]]
+then
+   echo "Config $CFG not found!"
+   exit 1
+fi
+
+./check_vlasiator_cfg.sh $BIN $CFG > check.txt
+
+if grep -q "Invalid options" check.txt 
+then
+    echo "Invalid options in config file, check check.txt"
+    echo ""
+    ERR=1
+fi
+
+SW=$(sed -n s/^file_..\\s*=\\s*//p < $CFG)
+
+if ! test -f $SW
+then
+    echo "Solar wind file $SW not found!"
+    echo ""
+    ERR=1
+fi
+
+MF=$(sed -n s/^atmosphericModelFile\\s*=\\s*//p < $CFG)
+if [[ -z $MF ]]
+then
+    MF=NRLMSIS.dat
+fi
+
+if ! test -f $MF
+then
+    echo "Atmospheric model file $MF not found!"
+    echo ""
+    ERR=1
+fi
+
+IO=$(sed -n s/^system_write_path\\s*=\\s*//p < $CFG)
```

Comment:
Same, does this check for several IO directories?

Reply:
Fixed to iterate over every IO path